### ISSUE TITLE
Customizable background

### ISSUE THREAD

**Issue by Lineflyer** [culmor30/cgeo-wear#15]
Feature request:

While the current default background is nice, still some people might prefer a customizable background, e.g. I would prefer black to make it more readable in sunlight (higher contrast).

**Comment by Robert136** [comment]
I agree... It would be a nice feature to be able to switch to a dark background...

By advance thanks!
